client/client.go

```diff
@@ -36,12 +39,18 @@ type Client struct {
 }
 
 // NewClient Return a new client with loaded configuration
-func NewClient(config Config) *Client {
+func NewClient(config Config) (*Client, error) {
 	httpClient := http.DefaultClient
 	if config.Auth.Enabled {
-		cert, err := tls.LoadX509KeyPair(config.Auth.CertPath, config.Auth.KeyPath)
+		var cert tls.Certificate
+		var err error
```

Comment:
this seems to be wrong

Reply:
Why are you redeclaring this var ?

Reply:
Because I'm using inside each block of the if/else.